docset/winserver2022-ps/smbshare/New-SmbGlobalMapping.md

```diff
@@ -0,0 +1,266 @@
+---
+description: Use this topic to help manage Windows and Windows Server technologies with Windows PowerShell.
+external help file: SmbGlobalMapping.cdxml-help.xml
+Module Name: SmbShare
+ms.date: 08/31/2018
+online version: https://docs.microsoft.com/powershell/module/smbshare/new-smbglobalmapping?view=windowsserver2022-ps&wt.mc_id=ps-gethelp
+schema: 2.0.0
+title: New-SmbGlobalMapping
+---
+
+# New-SmbGlobalMapping
+
+## SYNOPSIS
+Specifies Server Message Block (SMB) leasing and oplock behaviors.
+
+## SYNTAX
+
+```
+New-SmbGlobalMapping [[-LocalPath] <String>] [-RemotePath] <String> -Credential <PSCredential>
+ [-Persistent <Boolean>] [-RequireIntegrity <Boolean>] [-RequirePrivacy <Boolean>] [-UseWriteThrough <Boolean>]
+ [-FullAccess <String[]>] [-DenyAccess <String[]>] [-CimSession <CimSession[]>] [-ThrottleLimit <Int32>]
+ [-AsJob] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+The New-SmbGlobalMapping cmdlet creates an SMB global mapping on the SMB client to an SMB share. Global mappings allow all users to use the same mapping. Its primary use is for Windows Containers.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> $creds = Get-Credential New-SmbGlobalMapping -RemotePath \\fs1.contoso.com\public -Credential $creds -LocalPath G:
```

Comment:
Please add the output format here as you did with other examples.

Reply:
@nedpyle can you address this request?

Reply:
Email to Ned also sent.

Reply:
HI. I don't understand the request. The output format is already there below???

Status Local Path Remote Path 
------ ---------- ----------- 
OK     G:         \\fs1.contoso.com\public

Reply:
My bad Ned.  Apologies.  I thought it was missing.